sdk/eventhubs/azure-messaging-eventhubs/vcpkg/vcpkg.json

```diff
@@ -18,8 +18,13 @@
     {
       "name": "azure-core-amqp-cpp",
       "default-features": false,
-      "version>=": "1.0.0-beta.7"
+      "version>=": "1.0.0-beta.8"
     },
+    {
+      "name": "azure-core",
```

Comment:
We currently don't add explicit dependencies on Core, let's address it separately - #4795.
If you still want it, I think it makes sense to also update Config.camke.in to have `find_dependency(azure-core-cpp)`.

Reply:
How do we deal with packages that have a dependency on functionality which is only a specific core version if they're not allowed to have a dependency on core?